tests/integ/test_column_names.py

```diff
@@ -25,32 +25,49 @@
     upper,
     when,
 )
+from snowflake.snowpark.mock.connection import MockServerConnection
 from tests.utils import Utils
 
 pytestmark = pytest.mark.xfail(
     condition="config.getvalue('local_testing_mode')",
     raises=NotImplementedError,
-    strict=True,
+    strict=False,
 )
 
 
 def get_metadata_names(session, df):
+    if isinstance(session._conn, MockServerConnection):
+        return [col.name for col in session._conn.get_result_and_metadata(df._plan)[1]]
+
     description = session._conn._cursor.describe(df.queries["queries"][-1])
     return [quote_name(metadata.name) for metadata in description]
 
 
-def test_like(session):
-    df1 = session.sql("select 'v' as c")
```

Comment:
I think we can just replace the original line with ` session.create_dataframe`.

Reply:
you are right! updated